setup.py

```diff
@@ -6,8 +6,8 @@
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
-from distutils.core import setup
-from distutils.core import Extension
+from setuptools import setup
+from setuptools import Extension
```

Comment:
This works locally, but break the github build. Will remove this change and create
an issue to fix it later.